pymc3/step_methods/hmc/hmc.py

```diff
@@ -127,13 +131,13 @@ def _hamiltonian_step(self, start, p0, step_size):
             div_info = DivergenceInfo("Integration failed.", e, last, None)
         else:
             if not np.isfinite(state.energy):
-                div_info = DivergenceInfo("Divergence encountered, bad energy.", None, last, state)
+                div_info = DivergenceInfo("Divergence encountered: infinite energy.", None, last, state)
```

Comment:
Although this is a better description, we might want to stick with `bad` as it is easier for users to find out about this in search engines

Reply:
That's a fair point. I changed it because one of my undergrads was confused by the error message, and explaining them that the energy came out infinite cleared it up. But admittedly, that's n=1 usar evidence...

Reply:
I reverted the change.